fili-core/src/main/java/com/yahoo/bard/webservice/application/AbstractBinderFactory.java

```diff
@@ -754,6 +757,9 @@ protected final void setupDruidDimensionsLoader(
                 throw new IllegalStateException(e);
             }
         } else if (BardFeatureFlag.DRUID_CACHE.isOn()) {
+            LOG.warn("Cache V1 feature flag is deprecated, " +
+                    "use the new configuration parameter to set desired caching strategy"
+            );
```

Comment:
what about binding if cache v3 is set via the new configuration setting?  Under this code, the StubCache would be built, which I think is wrong.